Assets/LeapMotionModules/ElementRenderer/Resources/DynamicRenderer.cginc

```diff
@@ -0,0 +1,207 @@
+// Upgrade NOTE: replaced '_Object2World' with 'unity_ObjectToWorld'
+
+#include "Assets/LeapMotionModules/ElementRenderer/Resources/LeapGui.cginc"
+
+/***********************************
+ * Space name:
+ *  _ (none)
+ *    rect space, with no distortion
+ *  LEAP_GUI_CYLINDRICAL
+ *    cylindrical space
+ ***********************************/
+
+float4x4 _LeapGui_WorldToGui;
+float4x4 _LeapGui_GuiToWorld;
+
+#ifdef LEAP_GUI_CYLINDRICAL
+#define LEAP_GUI_WARPING
+#include "Assets/LeapMotionModules/ElementRenderer/Resources/CylindricalSpace.cginc"
+
+float4 _LeapGuiCurved_ElementParameters[ELEMENT_MAX];
+float4x4 _LeapGui_LocalToWorld;
+
+void ApplyGuiWarping(inout float4 anchorSpaceVert, int elementId) {
+  float4 parameters = _LeapGuiCurved_ElementParameters[elementId];
+
+  Cylindrical_LocalToWorld(anchorSpaceVert.xyz, parameters);
+
+  anchorSpaceVert = mul(_LeapGui_LocalToWorld, vert);
```

Comment:
This line is yelling at me when I try to switch to a dynamic renderer 😨

`Shader error in 'LeapGui/Defaults/DynamicTransparent': 'mul': no matching 2 parameter intrinsic function; Possible intrinsic functions are: mul( *lots of bullshit* )  at Assets/LeapMotionModules/ElementRenderer/Resources/DynamicRenderer.cginc(28) (on d3d11)`

`Shader error in 'LeapGui/Defaults/DynamicTransparent': undeclared identifier 'vert' at Assets/LeapMotionModules/ElementRenderer/Resources/DynamicRenderer.cginc(28) (on d3d11)`

Have you tried switching your build target back to PC, away from Android recently?